scripts/plot_radiosonde_data.py

```diff
@@ -0,0 +1,418 @@
+
+This script produces 3 plots:
+    - trajectory
+    - temperature T, dew point tau, relative humidity rh, water vapor mixing ratio, wind speed, and wind direction
+    - SkewT diagram (only if direction, i.e. ascent or descent, is given in inputfilename)
+"""
+
+import argparse
+import os.path
+
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import metpy.calc as mpcalc
+import numpy as np
+import xarray as xr
+from metpy.plots import Hodograph, SkewT
+from metpy.units import units
+from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-i",
+        "--inputfile",
+        metavar="INPUT_FILE",
+        help="Converted sounding file (netCDF)",
+        default=None,
+        required=True,
+    )
+
+    parser.add_argument(
+        "-o",
+        "--outdir",
+        metavar="/some/example/path/",
+        help="Output directory for plots",
+        default="/Users/laura/ownCloud/Campaigns/Radiosonden/plots",
+        required=False,
+    )
+
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        metavar="DEBUG",
+        help="Set the level of verbosity [DEBUG, INFO," " WARNING, ERROR]",
+        required=False,
+        default="INFO",
+    )
+
+    parsed_args = vars(parser.parse_args())
+
+    return parsed_args
+
+
+def main():
+    args = get_args()
+
+    # Define input file
+    filename = args["inputfile"]
+
+    data = xr.open_dataset(filename)
+    if "ascent" in filename:
+        direction = "ascent"
+    elif "descent" in filename:
+        direction = "descent"
+    else:
+        direction = "unknown"
+
+    quantities_color = "steelblue"
+
+    # Check and create out directories
+    outdir = args["outdir"]
+    check_outdir = os.path.exists(f"{outdir}/Trajectories")
+    if not check_outdir:
+        os.mkdir(f"{outdir}/Trajectories")
+    check_outdir = os.path.exists(f"{outdir}/Quantities")
+    if not check_outdir:
+        os.mkdir(f"{outdir}/Quantities")
+    check_outdir = os.path.exists(f"{outdir}/SkewT")
+    if not check_outdir:
+        os.mkdir(f"{outdir}/SkewT")
+
+    snd = 0  # sounding
+    launch_time = np.datetime64(data.launch_time.values[0], "m")
+    timestamp = str(np.datetime64(launch_time, "m")).replace(":", "")
+
+    props = dict(boxstyle="round", facecolor="white", alpha=0.8)
+
+    # Trajectory of the radiosonde
+    fig_track, ax = plt.subplots(figsize=(15, 5))
+    ax = plt.subplot(projection=ccrs.PlateCarree())
+    ax.coastlines(color="black")
+    ax.stock_img()
+    ax.gridlines(
+        crs=ccrs.PlateCarree(),
+        draw_labels=True,
+        linewidth=1,
+        color="gray",
+        alpha=0.5,
+        linestyle="--",
+    )
+    traj = plt.scatter(
+        data.isel(sounding=snd).lon,
+        data.isel(sounding=snd).lat,
+        c=(data.isel(sounding=snd).alt_WGS84.values) / 1000,
```

Comment:
```suggestion
        c=(data.isel(sounding=snd).alt_WGS84.pint.to('kilometer').pint.dequantify().values,
```
Allowing the input data to be of any compatible unit.